rlberry/manager/evaluation.py

```diff
@@ -304,6 +304,9 @@ def plot_writer_data(
     if ax is None:
         figure, ax = plt.subplots(1, 1)
 
+    # PS: in the next release of seaborn, ci should be deprecated and replaced
+    # with errorbar, which allows to specifies other types of confidence bars,
+    # in particular quantiles.
```

Comment:
Nice ! we may want to change to quantile then. As of now, we chose the 'sd' ci because choosing the permutation ci (ie argument ci = 75 for instance) is costly with large dataset.

Reply:
Absolutely, when it's out I'd much rather see quantiles (the update is already on github btw, just not released yet to the latest version of seaborn).